NebulaNetwork/Client.cs

```diff
@@ -246,7 +252,10 @@ private void ClientSocket_OnClose(object sender, CloseEventArgs e)
         private static void DisableNagleAlgorithm(WebSocket socket)
         {
             TcpClient tcpClient = AccessTools.FieldRefAccess<WebSocket, TcpClient>("_tcpClient")(socket);
-            tcpClient.NoDelay = true;
+            if (tcpClient != null)
+            {
+                tcpClient.NoDelay = true;
+            }
```

Comment:
was this done for a particular reason? Is tcpClient null when opening the connection for the lobby? i think it shouldn't be hm

Reply:
If it was for a particular reason i dont remember it anymore :)